daemon/config/config.go

```diff
@@ -275,3 +287,12 @@ func getConflictConfigurations(flagSet *pflag.FlagSet, fileFlags map[string]inte
 func mergeConfigurations(src *Config, dest *Config) error {
 	return utils.Merge(src, dest)
 }
+
+// validateCgroupDriver validates cgroup driver
+func validateCgroupDriver(driver string) error {
+	if driver == "" || driver == CgroupfsDriver || driver == CgroupSystemdDriver {
```

Comment:
when driver is empty, which one will be used?

Reply:
updated